CMakeLists.txt

```diff
@@ -83,6 +85,9 @@ include_directories( ${svlIncludeDir} )
 #==================================================================
 add_subdirectory( IbisITK )
 add_subdirectory( IbisVTK )
+if( DEFINED OpenCV_DIR )
```

Comment:
The fact that the variable is defined doesn't necessarily mean the lib was found. Use something like if( OPENCV_FOUND ) instead.